addons/nodot/interaction/Portal3D.gd

```diff
@@ -0,0 +1,67 @@
+#Write by: Z3R0_GT 
+#Time: 04/19/23
+#Last update: 04/19/23
+
+##Portal3D node
+class_name Portal3D extends Area3D
+
+@export_category("Nodes")
+##Objective to teleport
+@export_node_path("Node3D") var Target
+##Objective for teleport
+@export_node_path("CharacterBody3D") var Player
+
+##Size of collision area
+@export_category("Size collision")
+@export var X : float = 3.372
+@export var Y : float = 3.821
+@export var Z : float = 1
+
+#Get position of "Target" and "Player"
+@onready var posPlayer = get_node(Player)
```

Comment:
Can we use snake_case for var names? It's consistent with the rest of the library